sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/json/JsonSuite.scala

```diff
@@ -1913,6 +1913,44 @@ abstract class JsonSuite
     }
   }
 
+  test("SPARK-45035: json enable ignoreCorruptFiles/ignoreMissingFiles with multi-Line") {
+    withCorruptFile(inputFile => {
+      withSQLConf(SQLConf.IGNORE_CORRUPT_FILES.key -> "false") {
+        val e = intercept[SparkException] {
+          spark.read.option("multiLine", true).json(inputFile.toURI.toString).collect()
```

Comment:
Is there a lack of testing for `TextInputJsonDataSource`?

Reply:
already tested in `FileBasedDataSourceSuite`

Reply:
I can't find them. Could you paste the link?

Reply:
oh, sorry. I misread ignoreCorruptFiles and ignoreMissingFiles. Added.